pom.xml

```diff
@@ -124,6 +124,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
```

Comment:
This is necessary for using `@SuperBuilder`. It also forces us to use `@Jacksonize` on some forms, so we might want to revert it.

Other approach: tie this version to Spring Boot 2.5.1 right now and remove the version tag when #94 is merged.

Reply:
Ok, Spring Boot 2.6.8 is using this exact version. Let's just remember to remove that line in the merge commit.